### PR TITLE
Add information about RCJ 2022

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov <ado.matejov@gmail.com>

### CODE REVIEW

File: docs/world-competition.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# RoboCupJunior SoccerSim World Championship 2022
+
+RoboCupJunior World Championship 2022 will be held in July in Bangkok,
+Thailand. After last year’s success, we would like to continue with the
+organization of the SoccerSim challenge this year as well.
+The competition will be held virtually, but in an “offline” mode this time,
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
The competition will be held virtually, but in an "offline" mode this time,
```

File: docs/world-competition.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Competition overview
+
+SoccerSim challenge this year includes:
+* Simulation of matches
+* Presentation of team description papers
+* Solving Technical Challenges.
+
+As usual, teams will be asked to submit the code of their robot players.
+We have frozen the rcj-soccersim software which is [available as a tagged release
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
We have frozen the `rcj-soccersim` software which is [available as a tagged release
```

File: docs/world-competition.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+In order to spread the knowledge, we ask teams to provide team
+description papers. These might contain descriptions of team members’ roles,
+various approaches you have tried while implementing the code for
+simulated robots as well as anything you consider worth mentioning.
+For inspiration, feel free to
+[check last year's papers out](https://github.com/RoboCupJuniorTC/awesome-rcj-soccer#soccersim)!
+
+There will be at most two technical challenges.
+rcj-soccersim software will be updated appropriately for these challenges.
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
`rcj-soccersim` software will be updated appropriately for these challenges.
```

File: docs/world-competition.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+There will be several days reserved for you to implement your solution
+and submit recorded video. Committee will then review it and evaluate each team.
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
and submit a recorded video. The committee will then review it and evaluate each team on a rubric.
```

File: docs/world-competition.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+The rules are based on the regular RoboCupJunior Soccer rules, adapted to the
+simulation environment.
+
+You can find them at
+https://github.com/RoboCupJuniorTC/soccer-rules-simulation/blob/master/rules.pdf
+
+**Note**: The rules haven't changed much from the last year. Perhaps the most
+important changes include the change of the world's coordinate system and
+removal of exact GPS coordinates of the ball. Robots are forced to sense the ball
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
removal of exact GPS coordinates of the ball. Robots are now forced to sense the ball
```

File: docs/world-competition.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+using IR sesnsors. We encourage you to check out the sample Robot controllers
```
Comment (mrshu) [inline docs/world-competition.md]: @Adman perhaps a link would make sense here?

File: docs/world-competition.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+as well as the [Webots](https://cyberbotics.com/doc/reference/worldinfo) documentation.
+
+## Documentation
+
+We now have two fully fledged documentation articles that can help you get
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
We now have two full fledged documentation articles that can help you get
```

File: docs/world-competition.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+- [Inter-robot communication](https://robocupjuniortc.github.io/rcj-soccersim/communication_between_robots/)
+
+If you feel like there is something missing in the documentation, do not
+hesitate to suggest a change -- Pull Requests are very much welcome!
+
+## Schedule
+
+_Unless otherwise specified, all dates use the [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth) timezone_
+
+* **June 1st:** Open Tournament Submission Form
```
Comment (mrshu) [inline docs/world-competition.md]:
```suggestion
* **June 8th:** Open Tournament Submission Form
```

File: docs/world-competition.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+* **July 12th:** Publishing Solutions of Challenge 1 as Videos on the Forum
+* **July 12th:** Team Description Paper Deadline
+* **July 12th:** Challenge 2 Handout
+* **July 13th .. 16th:** Publishing Tournament Results
+* **July 15th:** Challenge 2 Deadline
+* **July 23th:** Publishing Solutions of Challenge 2 as Videos on the Forum
+
+## Registration
+
+Registration form will be published soon!
```
Comment (mrshu) [inline docs/world-competition.md]: @Adman I guess at this point we should simply add it in here.

**Review by mrshu — APPROVED** [review]
@Adman just a few small changes here and there -- thanks a bunch for this!

File: docs/world-competition.md (hunk 6 above)
Comment (Adman) [reply docs/world-competition.md]: Added

File: docs/world-competition.md (hunk 9 above)
Comment (Adman) [reply docs/world-competition.md]: Added

**Review by mrshu — APPROVED** [review]

**Pull request merged**